### PR TITLE
[Docs](bangc-ops): Added the API Guide structure

### PR DESCRIPTION
**Pull request by GraceGuanH**
This is the BANGC OPS doc structure.

### CODE REVIEW

**Review by WangYuan-cn — APPROVED** [review]

File: docs/bangc-docs/api_guide/h2rst/ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## v1.0.1
+
+* 日期: 2021-2-4
+
+### 新增特性
+1. c的api.rst文件中添加函数参数
+2. 同名函数放在同一个大纲下
+
+## v1.1.0
```
Comment (WangYuan-cn) [inline docs/bangc-docs/api_guide/h2rst/ReleaseNotes.md:40]: 该文件和当前仓库不匹配

File: docs/bangc-docs/api_guide/h2rst/config.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# api文件的一级标题是否支持引用标签
+api_title_link_label = True
+
+# 是否解析类的私有成员
+get_private = False
+
+# 是否解析类的保护成员
+get_protected = False
+
+# 是否解析宏定义
```
Comment (WangYuan-cn) [inline docs/bangc-docs/api_guide/h2rst/config.py:24]: 若脚本后续会有大的改动，把中文注释都删掉吧

Comment (GraceGuanH) [reply docs/bangc-docs/api_guide/h2rst/config.py:24]: 按照沟通的，中午注释保留不动哦

**Comment by GraceGuanH** [comment]
Resubmit a MR for this.

**Pull request closed without merging**